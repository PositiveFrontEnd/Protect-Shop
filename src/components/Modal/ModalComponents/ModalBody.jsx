import React from "react"
import PropTypes from 'prop-types'

const ModalBody = ({ children }) => {
    return (
        <div className="modal__container">
            <div className="modal__content modal__content-form">{children}</div>
        </div>
    )
}

ModalBody.propTypes = {
    children: PropTypes.any
}

export default ModalBody