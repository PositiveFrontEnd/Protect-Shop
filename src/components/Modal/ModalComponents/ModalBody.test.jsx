import React from "react";
import { fireEvent, getByText, render, screen } from "@testing-library/react";
import ModalBody from "./ModalBody";
import { BrowserRouter } from "react-router-dom";
import { Provider } from "react-redux";
import configureStore from "redux-mock-store";
describe("test Modal", () => {
    test("render Modal", () => {
        const mockStore = configureStore();
        const initialState = {

        }
        const children = "some text"
        const store = mockStore(initialState);
        const { asFragment } = render(
            <Provider store={store}>
                <BrowserRouter>
                    <ModalBody children={children} />
                </BrowserRouter>
            </Provider>
        );
        expect(asFragment()).toMatchSnapshot();
    });
});