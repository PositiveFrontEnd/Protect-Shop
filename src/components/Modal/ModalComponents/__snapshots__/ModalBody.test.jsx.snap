// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`test Modal > render Modal 1`] = `
<DocumentFragment>
  <div
    class="modal__container"
  >
    <div
      class="modal__content modal__content-form"
    >
      some text
    </div>
  </div>
</DocumentFragment>
`;
